### ISSUE TITLE
Python 3.10 ?

### ISSUE THREAD

**Issue by cl4k** [jterrace/pyssim#41]
Hello jterrace,

Thank you for your work,

I was on my way to use pyssim, however, i got an error while installing through pip.

`
collecting pyssim
  Using cached pyssim-0.5.tar.gz (2.3 MB)
  ERROR: Error [Errno 2] No such file or directory: '/usr/local/opt/python/bin/python3.7' while executing command python setup.py egg_info
  Preparing metadata (setup.py) ... error
ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: '/usr/local/opt/python/bin/python3.7'
`

But i think, this may be linked to python version as i'm using python 3.10, if this is the case, could you update you pypi package ?

Many thanks in advance

**Comment by jterrace** [comment]
The 0.5 version was released in June 2022 and should have had 3.10 support. I'm wondering if this is an environment issue on your side? I'm not seeing anything tying this to 3.7?

I just tried on my own virtualenv with 3.10 and it seemed to install fine:

```
~/pyssim-venv$ ./bin/python --version
Python 3.10.8
~/pyssim-venv$ ./bin/pip install pyssim
Collecting pyssim
....
Successfully built pyssim
Installing collected packages: pillow, numpy, scipy, pyssim
Successfully installed numpy-1.23.5 pillow-9.3.0 pyssim-0.5 scipy-1.9.3
```

**Comment by cl4k** [comment]
Thank you for the quick, i will try to find the error on my side.

Have a nice day
